cime_config/tests.py

```diff
@@ -230,12 +230,12 @@
     #e3sm MMF tests for development
     "e3sm_mmf_integration" : {
         "tests" : (
-            "ERP_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_fixed_subcycle",
-            "ERS_Ln9.ne4pg2_oQU480.F20TR-MMFXX.eam-mmf_use_VT",
-            "ERS_Ln9.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_use_ESMT",
-            "ERS_Ln9.ne4pg2_ne4pg2.F-MMFOMP.eam-single_thread",
-            "ERS_Ln9.ne4pg2_ne4pg2.F-MMF1-RCEMIP",
-            "SMS_Ln5.ne4_ne4.F-MMFXX-SCM-ARM97",
+            "ERP_Ln9.ne4pg2_ne4pg2.F-MMF1.eam-mmf_fixed_subcycle",
+            "ERS_Ln9.ne4pg2_oQU480.F-20TR-MMF1.eam-mmf_use_VT",
+            "ERS_Ln9.ne4pg2_ne4pg2.F-MMF1.eam-mmf_use_ESMT",
+            "ERS_Ln9.ne4pg2_ne4pg2.F-MMF1.eam-mmf_samomp",
+            "ERS_Ln9.ne4pg2_ne4pg2.F-RCE-MMF1",
+            "SMS_Ln5.ne4_ne4.F-MMF1-SCM-ARM97",
```

Comment:
SCP compset name does not match the one defined in config_compset. in config_compset, SCP compsets are F-SCM-MMF1-ARM97 & F-SCM-MMF1-RICO.

Reply:
Good catch!